src/main/java/frc/robot/commands/autos/components/GetAndScoreOpenSide.java

```diff
@@ -65,7 +67,6 @@ public GetAndScoreOpenSide(
     eventMap.put(
         "deployIntake",
         new IntakeSequence(intake, lift, lights)
-            .withTimeout(2.0)
```

Comment:
Why remove this? Don't we want a failsafe for the sensor?

Reply:
We have the retractIntake event now which serves the same purpose